### DISCUSSION TITLE
Install wikijs 3, node 18

### DISCUSSION THREAD

**Discussion by MasterHalm** [requarks/wiki#6426] [Help / Questions] [answered]
After downloaded, install wikijs3
and start node server
i have tgis error
Error: Failed to lookup view "error" in views directory "/home/user/wiki/server/views"
    at Function.render (/home/user/wiki/server/node_modules/express/lib/application.js:597:17)
How to solve this problem?

**Comment by NGPixel** [comment, marked as answer]
You're using an **unsupported** dev version.

Follow the [docs](https://docs.requarks.io/install) to install the stable version.
